### ISSUE TITLE
Don't compile their code to ES5 before publishing to npm

### ISSUE THREAD

**Issue by TungXuan** [OpusCapita/filemanager#205]
When build production version with `react-scripts build` I got the bellow error:
`Failed to minify the code from this file: 
./node_modules/@opuscapita/react-filemanager/lib/index.js:32612`

**Comment by kvolkovich-sc** [comment]
It strange.

I didn't use `react-scripts` before. But I tries:

```
npm init react-app . // using create-react-app
```

**App.js**

```
import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';
import { FileManager, FileNavigator } from '@opuscapita/react-filemanager';
import connectorNodeV1 from '@opuscapita/react-filemanager-connector-node-v1';

const apiOptions = {
  ...connectorNodeV1.apiOptions,
  apiRoot: `http://localhost:3020/api` // Or you local Server Node V1 installation.
};

const fileManager = (
 <div style={{ height: '480px' }}>
    <FileManager>
      <FileNavigator
        id="filemanager-1"
        api={connectorNodeV1.api}
        apiOptions={apiOptions}
        capabilities={connectorNodeV1.capabilities}
        initialResourceId={'Lw'}
        listViewLayout={connectorNodeV1.listViewLayout}
        viewLayoutOptions={connectorNodeV1.viewLayoutOptions}
      />
    </FileManager>
  </div>
);


class App extends Component {
  render() {
    return (
		<div className="App">
		  {fileManager}
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h1 className="App-title">Welcome to React</h1>
        </header>
        <p className="App-intro">
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>
      </div>
    );
  }
}

export default App;
```

```
yarn start
```

**Result:**

![screen shot 2018-05-14 at 8 59 50 pm](https://user-images.githubusercontent.com/24652543/40014862-612a3ce6-57ba-11e8-9a59-561c99fd2620.png)

**Comment by TungXuan** [comment]
Yes, you can run it with `yarn start`
But my issue I got when build a production version. you can try `yarn build`

**Comment by kvolkovich-sc** [comment]
I found this line.

See **line 32612:**

![screen shot 2018-05-15 at 10 13 17 am](https://user-images.githubusercontent.com/24652543/40042100-d5d7772a-5828-11e8-893f-b7e10591b851.png)

![screen shot 2018-05-15 at 10 13 40 am](https://user-images.githubusercontent.com/24652543/40042118-e8a3eb22-5828-11e8-89fc-a7a1f24c1bcf.png)

I replaced it and it build were successful, but I still don't understand where is a problem.

![screen shot 2018-05-15 at 10 13 25 am](https://user-images.githubusercontent.com/24652543/40042127-ee4eace2-5828-11e8-9aa7-8c2945b12903.png)

![screen shot 2018-05-15 at 10 13 47 am](https://user-images.githubusercontent.com/24652543/40042132-f2f630b2-5828-11e8-9b8e-b536a1df6033.png)

Any ideas?

**Comment by TungXuan** [comment]
Yes, I follow your steps but it will get many errors from your libs. How I can change it to es5 code?

**Comment by TungXuan** [comment]
I replace

**Comment by kvolkovich-sc** [comment]
Fixed in `@opuscapita/react-filemanager@1.0.12`.

@TungXuan, thanks again for a bug reporting! 

:beetle: :gun:
